Replicator/tests/ReplicatorCollectionSGTest.cc

```diff
@@ -553,6 +553,130 @@ TEST_CASE_METHOD(ReplicatorCollectionSGTest, "Push and Pull Attachments SG", "[.
     }
 }
 
+TEST_CASE_METHOD(ReplicatorCollectionSGTest, "Resolve Conflict SG", "[.SyncServerCollection]") {
+    string idPrefix = timePrefix();
+    constexpr size_t collectionCount = 1;
+    std::array<C4CollectionSpec, collectionCount> collectionSpecs = {
+        Roses
+    };
+    std::array<C4Collection*, collectionCount> collections =
+        collectionPreamble(collectionSpecs, "sguser", "password", kC4LogNone);
```

Comment:
I don't remember the detail of collectionPreamble() function, but what is the kC4LogNone for?

Reply:
This is for debugging purpose. If not None, this function will set the kSyncLog to that level. I probably should remove it.

Reply:
If all the tests are going to set it to None, it's not much useful. When debugging an issue, you can just enable the debug log for the test case-by-case.

Reply:
yes, I am going to remove it. It's mostly useful for my testing by per-case basis.